src/patternfly/_variables.scss

```diff
@@ -86,6 +86,7 @@
 
   // Gutter
   --pf-global--gutter: #{$pf-global--gutter};
+  --pf-global--gutter--md: #{$pf-global--gutter--md};
```

Comment:
In the documentation I think it would be worth noting the spacer size that the gutter uses and then how it changes on a medium breakpoint.

Reply:
You think this should be in the layouts' documents, or somewhere else?

Reply:
For example here in the Gallery Layout. "Adds space between children" is a little too broad and I think it would help if the consumer knew what size gutter was being applied.

<img width="977" alt="Screen Shot 2019-05-20 at 2 19 36 PM" src="https://user-images.githubusercontent.com/20118816/58043129-6b347480-7b0a-11e9-963f-45fa76e8c309.png">

Reply:
You may need to add it to every layout that uses the gutter

Reply:
What do you have in mind for documentation that? Not to say we shouldn't, but we don't really document spacing values currently. And if we update the value of the spacing, it would be good if we don't have to update every component/layout that uses it. I wonder if we could say it uses the globally defined gutter value and generally matches the gutters used for sections in the page layout?

Reply:
Ok yeah I get that we don't want to explicitly state the size but I think this will help to inform the user:
"Adds space between children by using the globally defined gutter value"

Reply:
Sounds good! done.